README.md

```diff
@@ -107,8 +107,108 @@ https://<your-username>.github.io/css-components/
 
 # Open your Pull Request
 
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+## ⚠️ Atenção

```

Comment:
@Pedriinn1110 The idea is to keep the official docs in English, if you want you could create a new folder called translates/ptbr and create a new readme.md with your translation, so you could link it inside the official readme inside a section called like `translations`

what do you think?